### DISCUSSION TITLE
Dacia Duster 2017 ESP issue

### DISCUSSION THREAD

**Discussion by MartinKogovsek** [cedricp/ddt4all#1323] [Q&A]
Hi,

I have a Dacia Duster 2017 imported from Germany (if it matters) and i would like to turn off ESP permanently if at all possible.

I turn it off with the button, but it turns back on after i go faster then 30km/h.

I havent done anything regarding car "programming" yet so i don't know anything, but for start id like to know if its even possible.

Regards, Thanks

**Comment by AndyIreland** [comment]
I havent got or had a Duster but basing if yours is a duster Phase 2 and not security gateway locked then you should be able to disable ASR by diagnostics in DDT4ALL :

![image](https://github.com/user-attachments/assets/8a55b984-68eb-4e7e-8292-4d798e8a4439)

when you do that though you will get error lights and beep and spanner come up on dash and warning and icon of car slipping on the dash , but normally by DDT4ALL you can disable ASR/ESP/ESP in the cluster TDB settings to stop that

**Reply by MartinKogovsek** [reply]
Thank you, is there any instructions about what i need to run & connect this software to my car?

Also, i looked at the fuse diagram and saw the "Stability control unit" would it be anything wrong if i just simply remove it?

**Reply by AndyIreland** [reply]
> Thank you, is there any instructions about what i need to run & connect this software to my car?

Yes loads of tutorials on youtube and other various websites:
here as example: https://youtu.be/H9WdfGMH85Y?si=S-wpOk1Dac986V89

if you dont know how to use it and what you need be very careful . DDT4ALL in wrong hands or people who dont know how to use it can be very dangerous for the car, especially if you are doing settings for the braking system! - and DDT4ALL can break your car if something goes wrong or you do something wrong!

**Reply by AndyIreland** [reply]
Also, i looked at the fuse diagram and saw the "Stability control unit" would it be anything wrong if i just simply remove it?

remove the fuse you mean? - You will almost get a warning on the dash about braking system error, car might even go into limp mode 'get me home mode' as well I am not sure.
